File: programmers/js_src/level01/87389.js
// 나머지 1이 되는 수 찾기

function solution(n) {
  var answer = 0;

  for (let i = 1; i < n; i++) {
    if (n % i == 1) {
      answer = i;
      break;
    }
  }

  return answer;
}

File: programmers/java/level01/87389.java
// 나머지가 1이 되는 수 찾기

class Solution {
    public int solution(int n) {
        int result = 0;
        
        for (int i = 2; i < n; i++) {
            if (n % i == 1) {
                result = i;
                break;
            }
        }
        
        return result;
    }
}